.github/workflows/model_uploader.yml

```diff
@@ -0,0 +1,412 @@
+name: Model Auto-tracing & Uploading
+on:
+  # Step 1: Initiate the workflow
+  workflow_dispatch:
+    inputs:
+      model_source: 
+        description: "Model source (e.g. huggingface)"
+        required: true
+        type: string
+        default: "huggingface"
```

Comment:
What does default mean here? Is this necessary? As this is required, may be we don't need any default value here?

Reply:
It will set default value like this. I feel like it's helpful for now because we only have huggingface model, so we don't need to type it everytime.
<img width="209" alt="Screenshot 2023-08-16 at 12 39 12 PM" src="https://github.com/opensearch-project/opensearch-py-ml/assets/106889996/9511d82e-7f60-427f-ac25-c5cfbdcf95cf">